components/mitsubishi_uart/mitsubishi_uart.cpp

```diff
@@ -119,6 +120,12 @@ void MitsubishiUART::update() {
     return;
   }
 
+  // Block until we get our extended capabilities information.
+  if (!_capabilitiesCache.has_value()) {
+    IFACTIVE(hp_bridge.sendPacket(ExtendedConnectRequestPacket::instance());)
+    return;
+  }
+
```

Comment:
Let's not block on this *at least* until we actually do something with it; maybe just put this in as a `TODO:`.

Depending on the implementation we may not ever have to block, but no need to preemptively do so.